llmspec/llmspec.py

```diff
@@ -121,7 +122,7 @@ def from_message(
         return cls(
             id=str(uuid.uuid4()),
             object="completion",
-            created=datetime.now(),
+            created=int(time.time() * 1000),
```

Comment:
Shall we use `second` or `millisecond`?

Reply:
Seconds not millis. Sorry I had my wires crossed when I was trying to work this out.

Reply:
Thanks. I guess you've been busy recently. Let me help you improve this PR.